src/components/RegistrationForm.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { Button, Form, Container, Message } from "semantic-ui-react";
+import { signUp } from "../modules/auth";
+import { useDispatch, useSelector } from "react-redux";
+import { useHistory } from "react-router-dom";
+
+const RegistrationForm = () => {
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const message = useSelector((state) => state.message);
+
+  return (
+    <Container>
+      <Form
+        data-cy="registration-form"
+        onSubmit={(event) => signUp(event, dispatch, history)}
+      >
+        <Form.Input
+          data-cy="email"
+          icon="user"
+          iconPosition="left"
+          label="Email:"
+          placeholder="email"
+          name="email"
+          type="email"
+        />
+
+        <Form.Input
+          data-cy="password"
+          icon="lock"
+          iconPosition="left"
+          placeholder="password"
+          label="Password:"
+          type="password"
+          name="password"
+        />
+
+        <Form.Input
+          data-cy="password-confirmation"
+          icon="lock"
+          iconPosition="left"
+          placeholder="confirm password"
+          label="Confirm Password:"
+          type="password_confirmation"
```

Comment:
please change this to `type='password'` so that the characters are hidden when typing. The type of password confirmation doesn't exist

Reply:
fixed